### PR TITLE
Shirley google sheets api

### PR DESCRIPTION
**Pull request by shirleymango**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- adding GET request with Google Sheets API

* **What is the current behavior?** (You can also link to an open issue here)
- no API usage in app

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- nope!

* **Other information**:

### CODE REVIEW

File: metau-capstone/Info.plist [hunk 1]
```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>sheets_api_key</key>
```
Comment (zenangao-fb) [inline metau-capstone/Info.plist:5]: nit: to make these variable clear, name it to `google_sheets_api_key`

File: metau-capstone/API/APIManager.m [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    
+    NSString *path = [[NSBundle mainBundle] pathForResource: @"Info" ofType: @"plist"];
+    NSDictionary *dict = [NSDictionary dictionaryWithContentsOfFile: path];
+    self.APIkey = [dict objectForKey: @"sheets_api_key"];
+    NSLog(@"API: %@", self.APIkey);
+    
+    return self;
+}
+
+- (void) getSheetsData: (void(^)(NSError *error))completion {
```
Comment (zenangao-fb) [inline metau-capstone/API/APIManager.m:37]: when design api interface to be invoked by other callsites, often worth to consider what is expected to return. In this case you likely need to handle the output of sheet api as well as error if any.
Couple ways to handle this:
1. have separate blocks handler; one for success case and one for error case
2. have a single handler that handle a response object which contains both the data & error; on success case the data part is non-nil and error would be nil; and vice versa for the the error case

File: metau-capstone/Info.plist (hunk 1 above)
Comment (shirleymango) [reply metau-capstone/Info.plist:5]: done!

File: metau-capstone/API/APIManager.m (hunk 2 above)
Comment (shirleymango) [reply metau-capstone/API/APIManager.m:37]: done! I updated the completion handler to contain both the data and error

**Pull request merged**